hl_extractor/hl_calc.py

```diff
@@ -217,9 +221,11 @@ def main(num_threads):
             else:
                 break
 
-if __name__ == "__main__":
+
+@app.cli.command()
+@click.option("--threads", "-t", default=DEFAULT_NUM_THREADS, show_default=True, help="Number of threads to start")
+def hl_calc(threads):
```

Comment:
What about making commands part of the top level manage.py? That way we only need to set one environment variable in order to run all commands.
Here's a proposal for a command hierarchy, I'm open to any other suggestions:

* flask run
  * flask ab init_db
  * flask ab compute_stats
  * flask ab add_admin   (all utils in manage.py go under some common commandclass)
    * flask ab dump full_db   (db command class goes under `ab`, or as `abdump`?)
    * flask ab util hl_cal
    * flask ab util dataset_eval  (not sure if additional apps should go under their own commandclass, or at toplevel `ab`)